pkg/meta/snap.yaml

```diff
@@ -1,5 +1,5 @@
 name: snapweb
-version: "0.26.9"
+version: "0.26.11"
```

Comment:
Why are we going from .9 to .11 and leaving out .10?

Reply:
The problem here is that there is a mirror of this in launchpad, used for deployment. And there, the built version is 0.26-10

Reply:
in any case I've replaced . by -

Reply:
Thanks!